src/screens/Ticketing/Tickets/RecentTickets/RecentTickets.tsx

```diff
@@ -64,6 +64,9 @@ export const RecentTickets = () => {
     <View
       style={{backgroundColor: theme.static.background.background_1.background}}
     >
+      <ThemeText type="body__secondary" style={styles.header}>
+        {t(RecentTicketsTexts.repeatPurchase.label)}
+      </ThemeText>
```

Comment:
Now this heading is shown also when there is no recent purchases, which will probably look a little odd. Maybe move it below into the `!loading && !!recentTickets.length` check?

Reply:
Oopsie..ofcourse :) Fixed.